modules/azurerm/sonar-base-instance/main.tf

```diff
@@ -100,11 +100,10 @@ resource "azurerm_user_assigned_identity" "dsf_base" {
   location            = var.resource_group.location
 }
 
-data "azurerm_subscription" "subscription" {}
+data "azurerm_subscription" "current" {}
 
 resource "azurerm_role_assignment" "dsf_base_storage_role_assignment" {
-  count                = var.binaries_location.az_resource_group != "" ? 1 : 0
```

Comment:
-Why was this count removed? It was added in this PR - https://github.com/imperva/dsfkit/pull/344- as a requirement for Sonar testing

Reply:
Seems like it was an accident. Sorry.
The 2 PRs were merged not to far from each other 4 months ago. Not sure I understand the logic. Can you re-fix it?

Reply:
I see, I'll get a new PR up when I get a chance.